transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/DefaultMigrationManager.java

```diff
@@ -60,7 +60,14 @@
                   Dialect.POSTGRESQL_9,
                   "ALTER TABLE TXNO_OUTBOX RENAME COLUMN blacklisted TO blocked",
                   Dialect.H2,
-                  "ALTER TABLE TXNO_OUTBOX RENAME COLUMN blacklisted TO blocked")));
+                  "ALTER TABLE TXNO_OUTBOX RENAME COLUMN blacklisted TO blocked")),
+          new Migration(
+              7,
+              "Add lastAttemptTime column to outbox",
+              "ALTER TABLE TXNO_OUTBOX ADD COLUMN lastAttemptTime TIMESTAMP(6) NULL AFTER invocation",
```

Comment:
got caught out by: https://dev.mysql.com/doc/refman/5.7/en/timestamp-initialization.html 
*explicit null declaration required on new column when dealing with mysql5 otherwise auto-update is in effect.